src/components/Api.js

```diff
@@ -0,0 +1,115 @@
+import { data } from "autoprefixer";
+
+class Api {
+  constructor(initialCards) {
+    this._initialCards = initialCards;
+  }
+
+  handleServerResponse(response) {
+    if (response.ok) {
+      return response.json();
+    }
+    return Promise.reject(`Error: ${response.status}`);
+  }
+
+  getInitialCards() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/cards", {
+      headers: {
+        authorization: "07909f6e-76be-4aa7-8439-3e97a34a8c13",
```

Comment:
According to the task you need to pass full `headers` into the constructor of `Api`. See here https://snipboard.io/pRdaAZ.jpg . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)